### PR TITLE
Adjusted the number of lines needed to insert package end tag

### PR DESCRIPTION
**Pull request by trocolit2**
@doudou, Could you review this pull request?

### CODE REVIEW

**Comment by trocolit2** [comment]
@doudou, I see.
But the current patch is the one available on master branch?
I checked the branches and i did not find other path more updated.

File: patches/apriltags.patch [hunk 1]
```diff
@@ -43,7 +43,7 @@ diff -urN ../apriltag-2015-03-18/CMakeLists.txt ./CMakeLists.txt
 diff -urN ../apriltag-2015-03-18/manifest.xml ./manifest.xml
 --- ../apriltag-2015-03-18/manifest.xml	1970-01-01 01:00:00.000000000 +0100
 +++ ./manifest.xml	2015-04-30 17:20:35.639306306 +0200
-@@ -0,0 +1,12 @@
+@@ -0,0 +1,14 @@
 +<package>
```
Comment (trocolit2) [inline patches/apriltags.patch]: I just check the number of line below the differ header and is bigger than 12.
This patch header is wrong anyway.

**Comment by 2maz** [comment]
Did encounter this as well with 18.01 and I agree with @trocolit2  that the current patch is wrong.
But the intended change is also incorrect,  there are 13 lines added, not 14.

**Comment by trocolit2** [comment]
@2maz, 13 or 14 lines added will work. But the strange thing is that on 16.04 works perfectly, but on 18.04 this error occur. Maybe the library used to perform the patch changes its behavior.

**Comment by 2maz** [comment]
Closed by #177

**Pull request closed without merging**